traveler-swift-core/TravelerKit/TravelerRoute.swift

```diff
@@ -56,7 +56,7 @@ extension PassengerRoute: Route {
         }
 
         switch errorCode {
-        case 2012:
+        case 2012, 2008:
```

Comment:
how come we have two here?

Reply:
Please add a bit of inline documentation for other developers to know whats going on in cases where its code base or something specific...Knowledge should be captured